### PR TITLE
Implement specified help commands #219

### LINKED ISSUE #219: [Enhancement] Help message for individual commands
# Current Implementation
Currently, help just shows a summary of commands and instructions for them

# Propose Enhancement
Allow users to type help <command> which will display specific instructions for that module
eg. `help addmod` will show detailed instructions on how to use the addmod command

### PR DESCRIPTION
**Pull request by T-Wan-Lin**
Closes #219

### CODE REVIEW

File: src/main/java/seedu/apollo/ui/Parser.java [hunk 1]
```diff
@@ -247,6 +254,20 @@ public static HelpCommand chooseHelpCommand(String helpCommandName) throws Illeg
...
+        case "bye":
+            return new ExitHelpCommand();
+        case "week":
+            return new WeekHelpCommand();
+        case "delmod":
+            return new DeleteModHelpCommand();
+        case "showmod":
+            return new ShowModHelpCommand();
+        case "addmod":
+            return new AddModHelpCommand();
```
Comment (yixuann02) [inline src/main/java/seedu/apollo/ui/Parser.java:270]: Good use of case statements.

**Review by yixuann02 — APPROVED** [review]
Looks good, good job.

File: src/main/java/seedu/apollo/command/utils/DeleteModHelpCommand.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package seedu.apollo.command.utils;
+
+import seedu.apollo.calendar.Calendar;
+import seedu.apollo.module.ModuleList;
+import seedu.apollo.storage.Storage;
+import seedu.apollo.task.TaskList;
+import seedu.apollo.ui.Ui;
+
+public class DeleteModHelpCommand extends HelpCommand{
```
Comment (PoobalanAatmikaLakshmi) [inline src/main/java/seedu/apollo/command/utils/DeleteModHelpCommand.java:9]: Good job on adding separate classes for each help command. Good OOP!

File: src/main/java/seedu/apollo/ui/Parser.java (hunk 1 above)
Comment (PoobalanAatmikaLakshmi) [reply src/main/java/seedu/apollo/ui/Parser.java:270]: Perhaps there is no need for the case for help? The parent helpcommand does have a execute function that prints out the full help message

**Review by PoobalanAatmikaLakshmi — COMMENTED** [review]
Overall good job!

**Pull request merged**